internal/gok/run.go

```diff
@@ -32,9 +32,12 @@ then it stores the program in RAM of a running gokrazy instance and runs the pro
 This enables a quick feedback loop when working on a program that is running on gokrazy,
 without having to do a full gok update every time you only want to update a single program.
 
+Command line arguments to the program will be combined from the instance configuration file
+and the ` + "`gok run`" + ` command line, in that order.
+
 Examples:
   % cd ~/go/src/github.com/stapelberg/scan2drive/cmd/scan2drive
-  % gok -i scan2drive run
+  % gok -i scan2drive run -- -tls_autocert_hosts=scan.example.com
```

Comment:
Can you leave both examples in and add a comment in front of the new example:

```
# specify extra flags on the command line
```

(See `gok help add` for how multiple examples should be formatted.)